### ISSUE TITLE
An option to use opts.file rather than opts.data

### ISSUE THREAD

**Issue by danreeves** [dlmanning/gulp-sass#324]
It appears custom imports don't currently play nicely with `opts.data`, and it doesn't pass the correct values to the importer. It works fine with `opts.file` however. 

It didn't like this option was already there. Any way to do this or any reason it shouldn't be there?

e.g.
With the code:
```js
return gulp.src(sassGlob)
.pipe(sass({
    importer: function (url, prev, done) {
        console.log(url, prev, done)
    }
}))
```
It should output
```js
// url: modules/* 
// prev: /path/to/your/assets/css/main.scss 
// done: function done(data) {
//        bridge.success(data);
//       }
```
but it outputs:
```js
// url: modules/* 
// prev: stdin 
// done: function done(data) {
//        bridge.success(data);
//       }
```

**Comment by danreeves** [comment]
https://github.com/danreeves/gulp-sass/commit/42338402710e1ce4db1a76667b97260415bc742c is a working fix with all tests passing. Let me know what you think.

**Comment by xzyfer** [comment]
Addressed by #353
